data_diff/database.py

```diff
@@ -163,8 +250,20 @@ def close(self):
 _CHECKSUM_BITSIZE = CHECKSUM_HEXDIGITS << 2
 CHECKSUM_MASK = (2**_CHECKSUM_BITSIZE) - 1
 
+DEFAULT_PRECISION = 6
+
+TIMESTAMP_BASE_LEN = 20
```

Comment:
Can you add a comment with how it's this bit:

```ruby
"2022-06-03 12:24:35."
```

Reply:
Yeah np